spec/requests/authors_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Authors', type: :request do
+  describe 'GET /authors/:id' do
+    context 'when authors exist' do
+      let(:author) { create(:author, name: 'Alexandre Versignassi') }
+
+      before { get author_path(author.id) }
+
+      it 'returns the author' do
+        expect(response).to have_http_status :ok
+        expect(JSON(response.body)['name']).to eq 'Alexandre Versignassi'
```

Comment:
Tá certo esse JSON aqui sem o `.parse`? Realmente precisa dele? Não seria melhor isso aqui:

```suggestion
        expect(JSON.parse(response.body['name']).to eq 'Alexandre Versignassi'
```
Veja ai se funciona e o que prefere usar, se funcionar dos dois jeitos, tudo bem, escolha uma jeito e usa de boas.